src/02_spec/07_modules/cdm/datainterface.rst

```diff
@@ -29,8 +28,13 @@ The following fields in the header of the DI packet are set:
   * - 0
     - ``reason``
    
-      Indicates the cause of the event generation. 
-      It can be a breakpoint or watchpoint hit. 
+      Indicates the cause of the event generation.
+      It can be a breakpoint or watchpoint hit.
+      Bits 16-15 indicates the state of the event generation.
+      '10' indicates watchpoint hit and '11' indicates breakpoint hit.
```

Comment:
"Bits 16-15 indicates the state of the event generation." What is the "state of the event generation"? 
Can I somehow see which of the configured breakpoints or watchpoints has been hit? Just knowing "it was any watchpoint" is typically not very helpful.

Reply:
We can surely add the watchpoint/breakpoint address (using DVR/DCR pair) in the reason payload word by increasing the number of payload words. But I think it will be good to keep the things minimal in the beginning. For now, just informing the debugger about the debug event should be fine. Once we are successful in achieving this task, we can definitely improve the specification.